tools-v2/pkg/cli/command/curvebs/query/chunk/chunkserver.go

```diff
@@ -115,15 +134,33 @@ func (cCmd *ChunkServerListInCoysetCommand) Print(cmd *cobra.Command, args []str
 }
 
 func (cCmd *ChunkServerListInCoysetCommand) RunCommand(cmd *cobra.Command, args []string) error {
-	result, err := basecmd.GetRpcResponse(cCmd.Rpc.Info, cCmd.Rpc)
-	if err.TypeCode() != cmderror.CODE_SUCCESS {
-		return err.ToError()
+	var infos []*basecmd.Rpc
+	var funcs []basecmd.RpcFunc
+	for _, rpc := range cCmd.Rpc {
+		infos = append(infos, rpc.Info)
+		funcs = append(funcs, rpc)
+	}
+	results, errs := basecmd.GetRpcListResponse(infos, funcs)
+	if len(errs) == len(infos) {
+		mergeErr := cmderror.MergeCmdErrorExceptSuccess(errs)
+		return mergeErr.ToError()
 	}
-	cCmd.Response = result.(*topology.GetChunkServerListInCopySetsResponse)
-	if cCmd.Response.GetStatusCode() != int32(statuscode.TopoStatusCode_Success) {
-		retErr := cmderror.ErrGetChunkServerListInCopySets(statuscode.TopoStatusCode(cCmd.Response.GetStatusCode()),
-			cCmd.Rpc.Request.GetLogicalPoolId(), cCmd.Rpc.Request.GetCopysetId())
-		return retErr.ToError()
+
+	key2Location := make(map[uint64][]*common.ChunkServerLocation)
+	cCmd.key2Location = &key2Location
+	for i, result := range results {
+		logicalpoolid := cCmd.Rpc[i].Request.GetLogicalPoolId()
+		copysetids := cCmd.Rpc[i].Request.GetCopysetId()
+		response := result.(*topology.GetChunkServerListInCopySetsResponse)
+		if response.GetStatusCode() != int32(statuscode.TopoStatusCode_Success) {
+			retErr := cmderror.ErrGetChunkServerListInCopySets(statuscode.TopoStatusCode(response.GetStatusCode()),
+				logicalpoolid, copysetids)
+			return retErr.ToError()
```

Comment:
Collect all response status exceptions, and then merge into one. Instead of returning when encountering one.